release-please-config.json

```diff
@@ -9,6 +9,12 @@
       "release-type" : "go",
       "tag-separator": "/",
       "versioning" : "default"
+    },
+    "api-js" : {
+      "package-name": "api-js",
+      "release-type" : "node",
+      "tag-separator": "/",
+      "versioning" : "default"
```

Comment:
Also wondering if the package-name is relevant. I think it's going to be used in the version tags created on the repo, but is that all?

Reply:
So, as an example.
```
    "packages/sdk/server-node": {},
```
The default release type is node.
The tag in this cause is determined by the package.json, but it removes the namespace. So `@launchdarkly/node-server-sdk` becomes `node-server-sdk`.

For go modules specifically I think I had to define a package-name because otherwise it did something crazy.

For any strategy, here `node` it updates all the relevant version configuration. For something like node it works magically because it is very uniform. So it can update the package.json. When using the node-workspace plugin it also can automatically update dependent packages. (Which is how js-core can just release everything in one PR.)